src/components/common/SideNavigation/index.tsx

```diff
@@ -0,0 +1,162 @@
+import { ReactComponent as WriteIcon } from 'assets/svg/common/write.svg';
+import { ReactComponent as MyPageIcon } from 'assets/svg/common/my-page.svg';
+import { ReactComponent as SearchIcon } from 'assets/svg/search/lens.svg';
+import { ReactComponent as StoreFrontIcon } from 'assets/svg/home/storefront.svg';
+import { ReactComponent as BookMarkIcon } from 'assets/svg/home/bookmark.svg';
+import { ReactComponent as GroupIcon } from 'assets/svg/home/group.svg';
+import { ReactComponent as ExpandIcon } from 'assets/svg/common/expand.svg';
+import { ReactComponent as FoldIcon } from 'assets/svg/common/fold.svg';
+import { useAuth } from 'store/auth';
+import cn from 'utils/ts/classNames';
+import useBooleanState from 'utils/hooks/useBooleanState';
+import { Link } from 'react-router-dom';
+import { useFilterFriend, useFilterNearby, useFilterScrap } from 'store/filter';
+import styles from './SideNavigation.module.scss';
+
+export default function TopNavigation(): JSX.Element {
+  const auth = useAuth();
+  const [visible, , , toggle] = useBooleanState(false);
+  const { filterFriendState, setFilterFriend } = useFilterFriend();
+  const { filterScrapState, setFilterScrap } = useFilterScrap();
+  const { filterNearbyState, setFilterNearby } = useFilterNearby();
+
+  const TABS = [
+    {
+      name: '',
+      icon: <LogoIcon />,
+      link: '/',
+
+    },
+    {
+      name: '검색',
+      icon: <SearchIcon />,
+      link: '/',
+
+    },
+    {
+      name: '글쓰기',
+      icon: <WriteIcon />,
+      link: '/post',
+    },
+    {
+      name: '마이페이지',
+      icon: <MyPageIcon />,
+      link: auth ? '/profile' : '/login',
+    },
+    {
+      name: '설정',
+      icon: <SettingIcon />,
+      link: '/setting',
+    },
+  ];
+
+  return (
+    <div>
+      <nav className={styles['side-navigation']}>
+        <ul className={styles['side-navigation__lists']}>
+          {TABS.map((tab) => (
+            <li
+              key={tab.name}
+              className={cn({
+                [styles['side-navigation__list']]: true,
+                [styles['side-navigation__list--clicked']]: tab.name === '검색' && visible,
+              })}
+            >
+              {tab.name === '검색' ? (
+                <button
+                  type="button"
+                  className={styles['side-navigation__button']}
+                  onClick={toggle}
+                  tabIndex={0}
+                >
+                  <div>{tab.icon}</div>
+                  <div>{tab.name}</div>
+                </button>
+              ) : (
+                <Link to={tab.link} className={styles['side-navigation__link']}>
+                  <div>{tab.icon}</div>
+                  <div>{tab.name}</div>
+                </Link>
+              )}
+            </li>
+          ))}
+        </ul>
+        <ul className={styles['bottom-navigation']}>
+          {auth ? (
+            <li className={styles['bottom-navigation__login']}>
+              <div>사용자</div>
```

Comment:
닉네임과 account중 고민하면서 아직 넣지 못한걸까요? ㅋㅋㅋ
닉네임이 되어야할 것 같긴 한데 임시 닉네임 길이가 너무길어서...ㅋㅋㅋㅋㅋㅋ

```suggestion
              <div>{auth.account}</div>
```

Reply:
UI가 변경되어 로그아웃 부분만 넣었습니다. 사용자 프로필의 경우 'user/me' api 쪽을 수정해야할것 같습니다.
[d2bfb7d](https://github.com/BCSDLab/JJBAKSA_FRONT_END/pull/92/commits/d2bfb7d53baa1d134a208f815eb20d67d323d933)